scylla/tests/hygiene.rs

```diff
@@ -0,0 +1,64 @@
+#![no_implicit_prelude]
+
+extern crate scylla as _scylla;
+
+#[derive(
+    _scylla::macros::FromRow,
+    _scylla::macros::FromUserType,
+    _scylla::macros::IntoUserType,
+    _scylla::macros::ValueList,
+    PartialEq,
+    Debug,
+)]
+#[scylla_crate = "_scylla"]
+struct TestStruct {
+    a: ::core::primitive::i32,
+}
+
+#[test]
+fn test_rename() {
+    use _scylla::cql_to_rust::{FromCqlVal, FromRow};
+    use _scylla::frame::response::result::CqlValue;
+    use _scylla::frame::value::{Value, ValueList};
+
+    pub fn derived<T>()
+    where
+        T: FromRow + FromCqlVal<CqlValue> + Value + ValueList,
+    {
+    }
+    derived::<TestStruct>();
+}
+
+#[test]
+fn test_derives() {
+    use ::core::primitive::u8;
+    use ::std::assert_eq;
+    use ::std::option::Option::Some;
+    use ::std::vec::Vec;
+    use _scylla::_macro_internal::{CqlValue, Row, Value, ValueList};
+    use _scylla::cql_to_rust::FromRow;
+
+    let test_struct = TestStruct { a: 16 };
+    fn get_row() -> Row {
+        Row {
+            columns: ::std::vec![Some(CqlValue::Int(16)),],
```

Comment:
Nit: comma before the `]` is unnecessary

Reply:
Ok, I wonder why `cargo fmt` didn't catch that